### DISCUSSION TITLE
请问，尝试订阅微信公众号，但按照步骤微小领和搜狗公众号都显示没有订阅源，是怎么回事呢？

### DISCUSSION THREAD

**Discussion by Berlinzz** [DIYgod/RSSHub#12767] [Q&A]
纯小白，研究了很久也没研究明白，买过其他订阅服务但不刷新，所以用rsshub试一下，结果大失败。另外，想进tele的交流群但显示是私密群，请问需要满足什么条件呢？

**Comment by Berlinzz** [comment]
进群成功了，继续研究失败的原因。

**Reply by yangsaipp** [reply]
我也在研究订阅公众号，一直失败，你现在成功了吗？可以分享下

**Reply by Berlinzz** [reply]
没有呢，hub上的都不行。买的werss。
